ValidBookingCountDomain/src/test/java/com/skedgo/tripkit/validbookingcount/domain/GetRemoteValidBookingCountTest.kt

```diff
@@ -7,17 +7,17 @@ import com.nhaarman.mockitokotlin2.whenever
 import org.junit.Test
 import io.reactivex.Observable.empty
 import io.reactivex.Observable.just
-import com.skedgo.tripkit.account.domain.HasUserToken
+//import com.skedgo.tripkit.account.domain.HasUserToken
 
 class GetRemoteValidBookingCountTest {
   val validBookingCountRepository: ValidBookingCountRepository = mock()
-  val hasUserToken: HasUserToken = mock()
```

Comment:
These booking/user-token related tests are good to keep. Worthwhile looking into why they don't work anymore.